INSTALL.md

```diff
@@ -121,7 +121,55 @@ For bare-metal installations, the full instructions for setting up PostgreSQL, T
     LATEST_VERSION=$(curl -s https://api.github.com/repos/timescale/promscale/releases/latest | grep "tag_name" | cut -d'"' -f4)
     curl -L -o promscale "https://github.com/timescale/promscale/releases/download/${LATEST_VERSION}/promscale_${LATEST_VERSION}_Linux_x86_64"
     chmod +x promscale
-    ./promscale --db-name promscale --db-password promscale --db-user promscale --db-ssl-mode allow --install-extensions
+    ./promscale --db.name promscale --db.password promscale --db.user promscale --db.ssl-mode allow --startup.install-extensions --startup.upgrade-prerelease-extensions
+    ```
+## Compile From Source on MacOS(Darwin, ARM64)
+
+For bare-metal installations, the full instructions for setting up PostgreSQL, TimescaleDB, and the Promscale Extension are:
+
+1) Follow [this doc]([url](https://docs.timescale.com/install/latest/self-hosted/installation-macos/#install-self-hosted-timescaledb-using-homebrew)) and install Timescale(which also install Postgres)
```

Comment:
In general I wouldn't recommend installing postgres and timescaledb via homebrew. It does work to get stuff setup, but then homebrew updates the postgres version and suddenly you don't have timescaledb anymore.

Reply:
Does the behaviour of brew matches with apt/yum/def on Linux distros? Do you recommend any alternative to brew?

Reply:
I use [asdf](https://asdf-vm.com/guide/getting-started.html) for managing node, golang, rust and postgres versions in the past.

Reply:
> Does the behaviour of brew matches with apt/yum/def on Linux distros

I don't think so. It's a combination of two things:

1. homebrew installs postgres in a path corresponding to the version, and then sets up symlinks to those paths. As an example, on my machine:
```
> ls -l $(which pg_config)
lrwxr-xr-x 1 james 39 May 23 13:42 /opt/homebrew/bin/pg_config -> ../Cellar/postgresql/14.3/bin/pg_config
```

2. Timescaledb _isn't_ installed into the postgres directory by default, as a user you need to run `timescaledb_move.sh` yourself.

The combination of these things means that when a new version of PG is released, brew updates it, and switches symlinks, but timescaledb is not installed in the new postgres install.

I guess it is as simple as running `timescaledb_move.sh` after upgrading postgres.